app/controllers/company.controller.js

```diff
@@ -0,0 +1,55 @@
+const db = require('../models');
+
+const {
+  Company,
+  User,
+  Sequelize: { Op },
+} = db;
+
+// Create a company
+exports.createCompany = async (req, res, next) => {
+  const {
+    body: {
+      userId,
+      companyName,
+      website,
+      primaryColor,
+      secondaryColor,
+      address,
+      logo,
+      bio,
+    },
+  } = req;
+
+  User.findById(userId)
+    .then((user) => {
+      const ownerUserId = user.id;
+      Company.findOrCreate({
```

Comment:
i trust that this works already but i would add `return` in front of this, since it makes sense to return something in a promise. in this case, another promise